packages/components/src/SvgIcon/utils/createSvgIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { $Diff } from "utility-types";
+import SvgIcon from "../SvgIcon";
+import type { SvgIconProps } from "../SvgIcon";
+
+type IconProps = $Diff<
+  SvgIconProps,
+  {
+    children: any;
+  }
+>;
+
+/**
+ * Generates SVG Icons from paths and fragments.
+ */
+export default function createSvgIcon(
+  path: React.ReactNode,
+  displayName: string,
+): React.FC<IconProps> {
+  const Component = (props: IconProps, ref: any) => {
```

Comment:
Number 3 of 3 `any` types in this PR.

Reply:
Do we actually need to forward refs for our use case here?

Reply:
I wonder if it was copied over from MaterialUI? I'm not sure the use cases for forwarding ref to an icon 🤔

Reply:
Use case would be if someone wants to get a handle to the actual SVG element itself.

Since we're not doing that here (as in, none of the components in this repo that I see are capturing a ref) we don't really need it.